Description: Remove default value of AI extension related environment variables _PR_AI_API_KEY & _PR_AI_URL
Author: VZstless <vzstless@aosc.io>
Last-Update: 2025-12-13
--- a/module-request-ai/src/requests.rs
+++ b/module-request-ai/src/requests.rs
@@ -213,7 +213,7 @@
 				if let Some(key) = option_env!("_DEF_PR_AI_API_KEY") {
 					key.to_string()
 				} else {
-					"Y29uZ3JhdHVsYXRpb25zLCB5b3UgZm91bmQgdGhlIHNlY3JldCE=".to_string()
+					"".to_string()
 				}
 			}
 		};
@@ -227,7 +227,7 @@
 				if let Some(url) = option_env!("_DEF_PR_AI_URL") {
 					url.to_string()
 				} else {
-					"https://iff.envs.net/stream-completions.py".to_string()
+					"".to_string()
 				}
 			}
 		};
